src/pages/components/WhySysl/WhySysl.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import classnames from 'classnames';
+import styles from './styles.module.css';
+
+const badges = [
+    {
+        title: <>Guaranteed Consistency</>,
+        imageUrl: 'img/icon/garanteed_consistency.svg',
+        description: (<>Sysl is built for change. With Sysl, your specifications become the single source of truth. All the related system specifications, code, diagrams, documentations keep the same version - no more stale designs. Also, in unified good style.</>),
+    },
+    {
+        title: <>Quality  Built-in</>,
+        imageUrl: 'img/icon/Quality Built-in.svg',
+        description: (<>With Sysl, you can generate high-quality code, diagrams and documentation which align to style guide and best practices. Embed security in from the start.</>)
+    },
+    {
+        title: <>Flexible</>,
+        imageUrl: 'img/icon/Flexible.svg',
+        description: (<>Sysl's open-ended architecture makes it possible to generation code for any programming language, with out-of-the-box support for Java and Go.</>),
```

Comment:
```suggestion
        description: (<>Sysl's open-ended architecture makes it possible to generate code for any programming language, with out-of-the-box support for Go.</>),
```
We don't have any transforms for Java atm so probably safer to leave it out for now? (The old reljam stuff does Java codegen, but I don't think we want to advertise it)